Initial placements findings added (#362)
DFE-Digital/bat-design-history: app/posts/register-trainee-teachers/2021-05-01-initial-placement-findings.md

## What we learned

All the users we tested with understood what they were being asked to do in regards to providing placement details.

Most of the people we spoke to could provide information for the first placement when registering a trainee in September, but for some this meant changing their current work practices. 

> “We’d struggle with this, wouldn’t we? We’d have to go and get this information from our placement team.” 
> Round 6 P7 (Scitt)

> “If I’m doing it in August I’ll do “No, I’m doing it later”. If there are any changes I’d have to go back in and amend [them]. If I was keying in September I would put the first. I wouldn’t put the second as there might be changes in the Autumn. [I prefer] keying in the correct information rather than going in and changing it.” 
> Round 5 P3 (HEI, EY employment)

The rationale behind asking for placements is opaque and can lead to additional frustrations.

> “Compliance states that we need two, but why does the DfE need to know where the person is placed? They just need to know that they are placed?” 
> Round 6 P6 (SCITT)

> We found that further placements are often unknown at the point of requesting TRN.

Reviewer: ```suggestion
We found that further placements are often unknown at the point of requesting a TRN.
```

> All users feel confident that Register will enable them to change or finalise the placement info at a later point.

Reviewer: ```suggestion
All users felt confident that Register will enable them to change or finalise the placement info at a later point.
```

> > “I've always keyed in the number of days. I’ve never not. We have our year planned out so to me it doesn't make any difference as long as we can change. I’ve had to change things due to the pandemic.” 

Reviewer: ```suggestion
> “I’ve always keyed in the number of days. I’ve never not. We have our year planned out so to me it doesn't make any difference as long as we can change. I’ve had to change things due to the pandemic.” 
```

> Round 5 P3 (SCITT)

> “We had a couple of times when a trainee changed placements, I wonder how I would record that? I think I would just add another placement I’m guessing and change the previous placement to just 2 weeks?”
> Round 5 P6 (SCITT)

> We learned that providers’ approach to placements was more varied than we anticipated. We assumed that a trainee would have two linear placements and that terminology would be consistent across the ITT market. Neither turned out to be true and our initial designs did not fit with user needs.

Reviewer: I wonder if there's a more plain english way of referring to `ITT market`

Reviewer 2: ...... and that ITT provider terminology would be consistent.

Author: Agreed. Gregs recommendation added.

### Placement terminology

Terminology varies by provider with numerous terms being used to describe similar things.

There is the concept of a main location:

> * Main school or main setting 
> * Host school 
> * Training location

Reviewer: ```suggestion
* main school or main setting 
* host school 
* training location
```

> These locations are typically schoolswhere the SCITT administration is based.

Reviewer 3: ```suggestion
These locations are typically schools where the SCITT administration is based.
```

For provider-led trainees, this could also be their ‘training location’ - where the academic training takes place. In this case providers do not count time spent at this location towards a placement duration. 

However on early years routes the ‘main setting’ is where a trainee might have one or even two official placements, it could also be their employer. 	

> “People would be employed at a nursery or a young child setting and the program has three placements - two of them would be at their own setting, one at a different setting. Those would be spaced out throughout the year.” 
> R6 P1

Early years trainees and candidates do not go to schools. They go to private nursery businesses. So early years providers tend to refer to the placement locations and early years salaried course employers as ‘settings’. 

We identified commonly used terms by providers when referring to placements:

> * First placement
> * Placement A
> * Second placement
> * Placement B
> * Contrast placement
> * Temporary placement
> * Complementary placement
> * Phase B

Reviewer: ```suggestion
* first placement
* placement A
* second placement
* placement B
* contrast placement
* temporary placement
* complementary placement
* phase B
```

> Placements are mandatory and provide trainees with  versatile teaching experience. ‘Contrast’ and ‘complementary’ placements reflect that notion and offer the trainee exposure to a different context than their main location, or previous placements. For example, an inner city placement might be contrasted with a rural placement. A placement in an all-girls school might be followed by a placement in an all-boys setting.

Reviewer: ```suggestion
Placements are mandatory and provide trainees with versatile teaching experience. ‘Contrast’ and ‘complementary’ placements reflect that notion and offer the trainee exposure to a different context than their main location, or previous placements. For example, an inner city placement might be contrasted with a rural placement. A placement in an all-girls school might be followed by a placement in an all-boys setting.
```

>“Oh this fills me with absolute dread. Everybody goes to every different school. A host school then a different school then a different ... I'd have to do it three times. It's just more work.” 
> Round 5 P2 (SCITT)

### Confirm placement details

{{ appFigure({
  image: {
    file: "confirm-placements.png",
    alt: "Confirm the placement details you have added"
  }
}) }}

Many participants commented that they would like to see the name of the school to help confirm the URN is the correct one.

>“URNs are unreliable:
>Because a school has gone from school to trust etc. these numbers might change.” 
>Round 5 P4 (SCITT)

The current design displays placements in order of them being added to Register. If a trainee changed locations midway through their placement, the update will currently not appear in chronological order.

>“There are instances where the trainee doesn't suit the school. Looking at this I don't know how [to reflect that in the record] unless there is another option to add another school there could be two or three schools needed in the for instance.”  
>Round 5 P5 (SCITT)

## Next steps

### Iterate our designs

> Requiring the user to input  a specific start date and duration in weeks doesn't reflect how placements really happen and therefore not meet some of our user’s needs. 

Reviewer: ```suggestion
Requiring the user to input a specific start date and duration in weeks doesn’t reflect how placements really happen and therefore not meet some of our user’s needs. 
```

Additionally, due to the one-day-a-week pattern we observed with some EY providers, it would be more appropriate to ask them about duration in days, not weeks.

We need to decide if we want to prioritise details of each location or details around the time spent at each placement location. Collecting a combination of the two is an overly complex task.

### Consult data consumers
 
We need to better understand why our data consumers want to know this data/what exactly they are tracking. If they are interested in understanding how much time the trainees spend physically teaching in schools (as opposed to being trained) - our results might be skewed. 

## Design todos

* include a search that allows for school name, postcode or URN
* if we decide to pursue collect dates, we will also need to collect the year the placement started

> * give our users' confidence by adding a location name, URN and postcode on the confirmation page

Reviewer: ```suggestion
* give our users’ confidence by adding a location name, URN and postcode on the confirmation page
```

* there is no list of locations for EY placements in DTTP. Instead we need to collect the setting’s names (nursery, childminder, children's centre), postcode and potentially URNs as free text fields

## Concerns with collecting placement details

We have concerns about the implications of making the collection of placement details a required task.
 
Considering the frequency of location change for some trainees, our hunch is that this will:

* create a significant amount of additional work, becoming a burden for our users
* increase errors as users might forget about the updating the system with placement changes

> By acknowledging the scale of the burden we are knowingly compromising our users' experience.

Reviewer: ```suggestion
By acknowledging the scale of the burden we are knowingly compromising our users’ experience.
```

How might we make the process of adding placement data fit with the reality of how placements happen to reduce effort and minimise potential for error?

Asking for the placement data at trainee record setup allows for flexibility and for users to create a workflow that suits them. This creates additional complexity as not all placements are known at the time of record setup so in this scenario our users need to understand that they can do some of a task now, but will need to finish it off later.

In turn collecting placement details at time of QTS recommendation removes complexity when creating records yet would mean users spend significantly more time completing the task of recommending for qualifications.

> By exploring different ways of prompting outstanding actions we could change provider's behavior to record data changes as soon as they occur, minimising potential for error and reducing time needed to recommend a trainee for QTS.

Reviewer: ```suggestion
By exploring different ways of prompting outstanding actions we could change provider’s behaviour to record data changes as soon as they occur, minimising potential for error and reducing time needed to recommend a trainee for QTS.
```
